lib/chef/resource/group.rb

```diff
@@ -51,6 +51,9 @@ class Group < Chef::Resource
       property :non_unique, [ TrueClass, FalseClass ], default: false,
                description: "Allow gid duplication. May only be used with the Groupadd provider."
 
+      property :comment, String,
+               description: "Specifies a comment to associate with the local group."
+
```

Comment:
@kapil I think we need to add introduced here

Reply:
@Vasu1105 fixed with review comment.